data/json/monster_special_attacks/monster_attacks.json

```
@@ -63,7 +67,9 @@
     "hit_dmg_u": "The %1$s slams into you!",
     "hit_dmg_npc": "The %1$s slams into <npcname>!",
     "no_dmg_msg_u": "The %1$s tries to slam into you, but stumbles aside.",
-    "no_dmg_msg_npc": "The %1$s tries to slam into <npcname>, but fails to."
+    "no_dmg_msg_npc": "The %1$s tries to slam into <npcname>, but fails to.",
+    "dodgeable": true,
+    "blockable": true
```

```suggestion
    "blockable": false
```
I think a slam is actually something you shouldn't be able to block, given that most of our blocks are parries.

---

What would stepping aside and tripping them count as?

---

I agree, commited suggestion 😄